plugins/inputs/upsd/upsd.go

```diff
@@ -71,27 +73,45 @@ func (u *Upsd) gatherUps(acc telegraf.Accumulator, name string, variables []nut.
 	}
 
 	fields := map[string]interface{}{
-		"status_flags":            status,
-		"ups_status":              metrics["ups.status"],
-		"input_voltage":           metrics["input.voltage"],
-		"load_percent":            metrics["ups.load"],
-		"battery_charge_percent":  metrics["battery.charge"],
-		"time_left_ns":            timeLeftS * 1_000_000_000, //Compatibility with apcupsd metrics format
-		"output_voltage":          metrics["output.voltage"],
-		"internal_temp":           metrics["ups.temperature"],
-		"battery_voltage":         metrics["battery.voltage"],
-		"input_frequency":         metrics["input.frequency"],
-		"nominal_input_voltage":   metrics["input.voltage.nominal"],
-		"nominal_battery_voltage": metrics["battery.voltage.nominal"],
-		"nominal_power":           metrics["ups.realpower.nominal"],
-		"battery_date":            metrics["battery.mfr.date"],
+		"battery_date": metrics["battery.mfr.date"],
+		"status_flags": status,
+		//Compatibility with apcupsd metrics format
+		"time_left_ns": timeLeftS * 1_000_000_000,
+		"ups_status":   metrics["ups.status"],
```

Comment:
@srebhan should the option instead of being force_floats be force_types and we do something similar to the string values here (minus time_left_ns)?

Reply:
Well I think the string values are very unlikely to be delivered as something else, so I guess `force_floats` is good.